### PR TITLE
Ubuntu wiki files

### PR DESCRIPTION
**Pull request by EmreTech**
I have made instructions for installing Ubuntu on any MBP with the T2 chip.

### CODE REVIEW

File: docs/distributions/ubuntu/faq.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+This seems to happen with certain MacBooks because the GRUB bootloader installer tries accessing the efivars/nvram, which Apple doesn't allow and the installer doesn't know what to do.
+
+There is a way to stop this. Boot into the installation media with an External USB Keyboard plugged in. Press e when you selected the "Try Ubuntu without installing" option.
+
+Scroll to the bottom with the arrow keys, and type in ``efi=noruntime``. With the External keyboard, then press CTRL+X or F10 to boot into the Live Media. The installation should work fine now.
+
+This issue has occured for anyone on the 16,1 and maybe the 16,4.
+
+(Credits to Redecorating#0350 for this fix)
```
Comment (networkException) [inline docs/distributions/ubuntu/faq.md:11]: I'd use their github name here to keep it consistent with other credits

File: docs/distributions/ubuntu/faq.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# Making the GRUB Menu appear
+
+The GRUB bootloader by default turns off the GRUB Menu. This means you can't boot into Bootcamp Windows if it's installed. This can be easily fixed after Ubuntu is fully installed.
+
+In a Terminal in Ubuntu, type in ``sudo nano /etc/default/grub``. Change line ``GRUB_TIMEOUT_STYLE`` to ``GRUB_TIMEOUT_STYLE=MENU``. Save the file by doing CTRL+X, Y, then enter.
```
Comment (networkException) [inline docs/distributions/ubuntu/faq.md:17]: Even tho its nice to have documentation tell you exactly what to do, I'd expect someone who wants to install linux on their mac to be knowledgeable enough about linux to edit a file in their preferred editor.
In other wiki pages "edit file /path/to/file" is used instead of "type (nano / vim / whatever) /path/to/file)", also to keep it independent from the packages installed on a machine

File: docs/distributions/ubuntu/faq.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+We've now changed the GRUB Bootloader settings, but we now need to update GRUB to apply these changes. Type in ``sudo update-grub`` and hit enter. After the command is done, you're finished.
+
+(Also credits to Redecorating#0350 for telling me about this fix, marcosfad for documenting the fix)
```
Comment (networkException) [inline docs/distributions/ubuntu/faq.md:21]: Same here as mentioned above, also you could link marcosfad's github

File: docs/distributions/ubuntu/faq.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+4. Press `w` to write the partition and `y` to confirm
+5. If gdisk doesn't quit, press `q` to exit the command
+
+It should work now. Reboot into the GRUB Menu and try booting into Windows.
+
+(Credits to gbrow004 for documenting this fix on his [Gist](https://gist.github.com/gbrow004/096f845c8fe8d03ef9009fbb87b781a4#fixing-bootcampwindows))
+
+# Why isn't sound working?
+
+For some reason, the creator of the mbp-ubuntu iso can't get the sound working after the install. You'll have to do this manually.
```
Comment (networkException) [inline docs/distributions/ubuntu/faq.md:44]: I'd rephrase this to

`Due to issues in the mbp-ubuntu install process, there is no sound working after install. You'll have to set it up manually`

File: docs/distributions/ubuntu/installation.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Download the latest safe release
+
+For mbp-ubuntu, we recommend using the 5.7.19 kernel because 5.8 and newer kernels can have problems.
```
Comment (networkException) [inline docs/distributions/ubuntu/installation.md:3]: I'd stick closer to the phrasing from here https://github.com/marcosfad/mbp-ubuntu/pull/26#discussion_r550875065

File: docs/distributions/ubuntu/installation.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+[Download here](https://github.com/marcosfad/mbp-ubuntu/releases/tag/v20.04-5.7.19-1)
+
+The iso you should download depends ony your machine, but usually the normal mbp iso works fine.
```
Comment (networkException) [inline docs/distributions/ubuntu/installation.md:7]:
```patch
-depends ony your machine
+depends on your machine
```

**Review by networkException — CHANGES_REQUESTED** [review]
A few things here and there but overall good work :+1:

**Comment by EmreTech** [comment]
Alright, I applied those changes

**Review by EmreTech — COMMENTED** [review]
Finished changing up the document

**Review by networkException — APPROVED** [review]
Alright thank you for the changes
